package.json

```diff
@@ -85,6 +85,7 @@
     "types-bn": "^0.0.1",
     "types-ethereumjs-util": "^0.0.5",
     "typescript": "^2.4.1",
+    "web3_beta": "https://github.com/ethereum/web3.js#1.0",
```

Comment:
isn't there a more succinct way of pointing to a github repo from package.json? Something like `ethereum/web3.js#1.0`